.github/workflows/deploy-to-s3.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy To S3
+
+on:
+  push:
+    branches: [ official ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.9
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v4
+      with:
+        aws-access-key-id: ${{ secrets.CI_CD_AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.CI_CD_AWS_SECRET_ACCESS_KEY }}
+        aws-region: ${{ secrets.AWS_DEFAULT_REGION }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install wrc
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+        # install wkhtmltopdf
+        wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.4/wkhtmltox-0.12.4_linux-generic-amd64.tar.xz -O wkhtml.tar.xz && tar -xf wkhtml.tar.xz --strip-components=1 -C /usr/local
+    - name: Deploy to S3
+      run: |
+        outputdir=/tmp/regulations
+        mkdir -p $outputdir
+        wrc --target=json -o $outputdir .
+        wrc --target=html -o $outputdir .
+        wrc --target=pdf -o $outputdir .
+        # Update version
+        git_hash=$(git rev-parse --short "$GITHUB_SHA")
+        echo "$git_hash" > $outputdir/version
+        aws s3 sync $outputdir s3://wca-regulations/
```

Comment:
More nit about consistency: You're directly dialing `s3://wca-regulations/`, whereas in the wca-documents PR you're passing the bucket name as an env and then dialing `s3://$SOME_ENV_VAR`. Please decide on one approach and use it consistently across our build scripts :)

Reply:
yeah because in the other action I'm using .sh files. I'll just make it inline everywhere